setup.py

```diff
@@ -20,7 +20,7 @@
     install_requires=install_requires,
     entry_points={
         'gui_scripts': [
-            'appname=appname.__main__:main'
+            'opcpa-tpr-config=opcpa-tpr-config.__main__:main'
```

Comment:
Throughout the package, I believe you'll need the name to use underscores not dashes (i.e. `opcpa_tpr_config`).  The command `opcpa-tpr-config.__main__:main` basically performs a python import, and that disallows dashes.  

So basically `opcpa-tpr-config` --> `opcpa_tpr_config`